<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCutting</class>
 <widget class="QWidget" name="ImageCutting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>681</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>切图选项</string>
   </property>
   <widget class="QLineEdit" name="edit_width">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_width">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>width:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_begin">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_height">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_height">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>height:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selected">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>选图</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_setSavePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>设置保存路径</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_savePath">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>NULL</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
